### PR TITLE
Database queries

### PR DESCRIPTION
**Pull request by majakudlicka**

### CODE REVIEW

File: database/database_queries.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const dbConnection = require('./db_connection.js');
+
+data = {};
+
+data.getStudents = (term, callback) => {
+  dbConnection.query(`SELECT * FROM jobs
+                      WHERE CATEGORY = ${term}
```
Comment (des-des) [inline database/database_queries.js]: No no no, this is vulnerable to SQL injection attack!
```js
.query(`
  SELECT * FROM jobs
  WHERE CATEGORY = $1
  AND end_date < NOW()
 ORDER BY start_date;`, [term], function(err, res) {
```

File: src/components/Form_Register_Student.js [hunk 2]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
       <form className="form" onSubmit={this._onSubmit.bind(this)}>
         <ErrorMessage />
         <div className="form__field-wrapper">
-        <label className="form__field-label" htmlFor="First Name">Password</label>
-        <input className="form__field-input" id="First Name" type="First Name" value={this.props.data.password}
-         placeholder="First Name"  onChange={this._changePassword.bind(this)} />
+        <label className="form__field-label" htmlFor="First Name">First Name</label>
+        <input className="form__field-input"
+         id="First Name"
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: indent these one out from parent

File: src/components/Form.js [hunk 3]
```diff
@@ -7,6 +7,8 @@
  */
 
 import React, { Component } from 'react';
+import ErrorMessage from './ErrorMessage.js';
+const assign = Object.assign;
```
Comment (des-des) [inline src/components/Form.js]: delete this!

File: src/components/Form_Register_Student.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
-
+import { changeForm } from '../actions/AppActions';
+import LoadingButton from './LoadingButton.js';
+import ErrorMessage from './ErrorMessage.js';
 
 const assign = Object.assign;
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: delete this!

File: src/components/Form.js (hunk 3 above)
Comment (majakudlicka) [inline src/components/Form.js]: Deleting this breaks the code as we reference both things in the Form

Comment (majakudlicka) [inline src/components/Form.js]: I've found out by trying :)

File: src/components/Form_Register_Student.js (hunk 4 above)
Comment (majakudlicka) [inline src/components/Form_Register_Student.js]: All these properties are being referenced in the From, deleting these lines them breaks the app

File: src/components/Form.js (hunk 3 above)
Comment (des-des) [inline src/components/Form.js]: Ops no this is actually fine

File: src/components/Form_Register_Student.js (hunk 4 above)
Comment (des-des) [inline src/components/Form_Register_Student.js]: Yeah its fine being an idiot

**Pull request merged**